### PR TITLE
no-deeply-nested-dependent-keys-with-each: Fix documentation examples

### PR DESCRIPTION
**Pull request by Alonski**
The examples had code that wouldn't work :)

### CODE REVIEW

File: docs/rules/no-deeply-nested-dependent-keys-with-each.md [hunk 1]
```diff
@@ -18,11 +18,11 @@ Examples of **correct** code for this rule:
 
 ```js
 displayNames: computed('owners.@each.name', function() {
-  return this.owners.mapBy(owners, 'name');
+  return this.owners.mapBy('owner', 'name');
 }),
 
 owners: computed('todos.@each.owner', function() {
-  return this.todos.mapBy(todo, 'owner');
+  return this.todos.mapBy('todo', 'owner');
```
Comment (Turbo87) [inline docs/rules/no-deeply-nested-dependent-keys-with-each.md]:
```suggestion
  return this.todos.mapBy('owner');
```
(mapBy)[https://emberjs.com/api/ember/3.7/classes/EmberArray/methods/mapBy?anchor=mapBy] only takes a single parameter

Comment (Alonski) [reply docs/rules/no-deeply-nested-dependent-keys-with-each.md]: Woops! I try to fix an error and end up not fixing it all :)

File: docs/rules/no-deeply-nested-dependent-keys-with-each.md [hunk 2]
```diff
@@ -18,11 +18,11 @@ Examples of **correct** code for this rule:
 
 ```js
 displayNames: computed('owners.@each.name', function() {
-  return this.owners.mapBy(owners, 'name');
+  return this.owners.mapBy('owner', 'name');
```
Comment (Turbo87) [inline docs/rules/no-deeply-nested-dependent-keys-with-each.md]:
```suggestion
  return this.owners.mapBy('name');
```

**Comment by Turbo87** [comment]
thanks :)

**Comment by Alonski** [comment]
Always glad to help :)

**Pull request merged**